### PR TITLE
Feature/home page

### PR DESCRIPTION
**Pull request by BalajiHarinath**
Added home-page and rules-page for the quiz app. Please review

### CODE REVIEW

File: pages/rules/rules.css [hunk 1]
```diff
@@ -0,0 +1,61 @@
+.page-container {
+    height: 100vh;
```
Comment (Hamza-777) [inline pages/rules/rules.css:2]: Avoid using height if possible.

File: pages/rules/rules.css [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+}
+
+/*Nav bar*/
+.navbar {
+    padding: 1rem 3rem;
+    border-bottom: 1px solid var(--tertiary-color);
```
Comment (Hamza-777) [inline pages/rules/rules.css:8]: Primary and secondary is enough any further than that is confusing instead name the color with respective colorname.

**Review by Hamza-777 — APPROVED** [review]
Everything else looks good.

**Review by VLeads — APPROVED** [review]
Looks good to go

File: styles.css [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+
+/*Footer*/
+.footer{
+    border: 1px solid var(--tertiary-color);;
+    z-index: 1;
+}
+
+.footer-social-icons {
+	margin: 1rem auto;
+    padding-left: 0;
```
Comment (MayankKumar10) [inline styles.css]: define padding 0 and margin 0 in your reset CSS or HTML tag so you don't have to use it elsewhere

**Review by MayankKumar10 — APPROVED** [review]
Everything else looks good you can merge

File: styles.css (hunk 3 above)
Comment (BalajiHarinath) [reply styles.css]: Yes I have already added the reset css. I forgot to remove the padding here. Thanks for pointing this out.

File: pages/rules/rules.css (hunk 2 above)
Comment (BalajiHarinath) [reply pages/rules/rules.css:8]: okay will make this change. Thanks for reviewing.

**Pull request merged**